src/main/java/me/ShermansWorld/AlathraExtras/AlathraExtras.java

```diff
@@ -158,12 +166,14 @@ public void onEnable() {
         this.getServer().getPluginManager().registerEvents(new NPCListener(), this);
         this.getServer().getPluginManager().registerEvents(new TrapdoorListener(), this);
         this.getServer().getPluginManager().registerEvents(new FoodConsumeListener(), this);
+        this.getServer().getPluginManager().registerEvents(new FarmingListener(), this);
```

Comment:
This entire listener should be skipped at registration time, rather than inside the listener's executor method.

```suggestion
        this.getServer().getPluginManager().registerEvents(new FoodConsumeListener(), this);
        if (getServer().getPluginManager().isPluginEnabled("CustomFishing"))
            this.getServer().getPluginManager().registerEvents(new FarmingListener(), this);
```